### PR TITLE
docs: Device

### PR DESCRIPTION
**Pull request by Ombuweb**

### CODE REVIEW

**Review by ashutosh887 — APPROVED** [review]
LGTM

File: content/guide/core/device.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+title: Device
+---
+
+This class allows you to get information about the current device, such the name of the os, manufacturer, etc.
```
Comment (rigor789) [inline content/guide/core/device.md]:
```suggestion
The `Device` class (singleton) contains information about the current device, such as `os`, `manufacturer`, `uuid` and more.
```

File: content/guide/core/device.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+
+## Using the Device class
+
+To use the Device class, import it from `@nativescript/core`.
+```ts
+import { Device } from '@nativescript/core'
+```
```
Comment (rigor789) [inline content/guide/core/device.md]:
```suggestion
To use the Device class, import it from `@nativescript/core`.

```ts
import { Device } from '@nativescript/core'
```
```

File: content/guide/core/device.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+```ts
+const deviceType: string = Device.deviceType // Phone
+```
+
+### Getting the device manufacturer
+
+The device manufacturer is accessed using the [manufacturer](#manufacturer) property.
+
+```ts
+const manufacturer: string = Device.manufacturer //"Samsung"
```
Comment (rigor789) [inline content/guide/core/device.md]:
```suggestion
const manufacturer: string = Device.manufacturer // "Samsung"
```

File: content/guide/core/device.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+SDK version:
+```ts
+const sdkVersion: string = Device.sdkVersion // 33
+```
+
+### Getting the device uuid
+To access the device's universally unique identifier(uuid), use the [uuid](#uuid) property:
+
+```ts
+const uuid: string = Device.uuid //cb83c3347392bfe2
```
Comment (rigor789) [inline content/guide/core/device.md]:
```suggestion
const uuid: string = Device.uuid // cb83c3347392bfe2
```

File: content/guide/core/device.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+---
+
+### sdkVersion
+
+Gets the SDK version.
+
+---
+### uuid
+
+Gets the uuid(Universal Unique Identifier) for the device. If you need to receive the same uuid even after the application re-install, consider using a plugin that handles this.
```
Comment (rigor789) [inline content/guide/core/device.md]:
```suggestion
Gets the <abbr title="Universal Unique Identifier">uuid</abbr> for the device.

::: info Note
If you need to receive the same uuid after the application is re-installed, consider using a plugin instead.
:::
```

**Review by rigor789 — COMMENTED** [review]
A few small suggestions/fixes - will apply them shortly.

**Comment by ashutosh887** [comment]
Great @rigor789

**Review by ashutosh887 — APPROVED** [review]

**Comment by ashutosh887** [comment]
Can wel close this issue now @rigor789 ?

**Pull request merged**